data.sql

```diff
@@ -11,3 +11,39 @@ INSERT INTO animals (name, date_of_birth, weight_kg,neutered, escape_attempts) V
 ('Angemon','2005-06-12',-45.0,true,1),
 ('Boarmon','2005-06-07',20.4,true,7),
 ('Blossom','1998-10-13',17.0,true,3);
+
+INSERT INTO owners(full_name, age) 
+VALUES 
+('Sam Smith',34),
+('Jennifer Orwell',19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+INSERT INTO species(name)
+VALUES ('Pokemon'),('Digimon');
```

Comment:
- [ ] Again, let us insert our data in this way

 INSERT INTO species (name)
 VALUES ('Pokemon');

 INSERT INTO species (name)
 VALUES ('Digimon');

Reply:
:heavy_check_mark: Well resolved